### PR TITLE
Add exclusion array to jobManager.ts

### PR DESCRIPTION
**Pull request by casthewiz**
Adds an exclusion array to fix an issue where `mms-docs` was incompatible with deployment benchmark changes

### CODE REVIEW

**Review by caesarbell — APPROVED** [review]
LGTM

**Review by seungpark — APPROVED** [review]

File: src/job/jobManager.ts [hunk 1]
```diff
@@ -116,8 +116,9 @@ export class JobManager {
     try {
       this._jobHandler = null;
       if (job?.payload) {
+        const excludeRepoFromBenchmarks = ['mms-docs'].includes(job.payload.repoName);
```
Comment (rayangler) [inline src/job/jobManager.ts:119]: Do we know if this is limited to `mms-docs`, or should this be for any old-gen properties?

Comment (casthewiz) [reply src/job/jobManager.ts:119]: I'm not certain on if `ruby` or `php` driver docs will fail in the same way - I think those are the only old-gen exceptions, now that server manual isn't actively deploying docs built with old-gen?

Comment (rayangler) [reply src/job/jobManager.ts:119]: Gotcha. Down to keep it as-is and alter the array if needed

**Review by rayangler — APPROVED** [review]

**Pull request merged**